pkg/chains/signing/kms/kms.go

```diff
@@ -95,12 +98,27 @@ func NewSigner(ctx context.Context, cfg config.KMSSigner) (*Signer, error) {
 	// pass through configuration options to RPCAuth used by KMS in sigstore
 	rpcAuth := options.RPCAuth{
 		Address: cfg.Auth.Address,
-		Token:   cfg.Auth.Token,
+		//Token:   cfg.Auth.Token,
```

Comment:
Can we remove this instead of adding a comment ?

Reply:
removed.